static/js/home-app.js

```diff
@@ -4,72 +4,56 @@ $(document).ready(function(){
 });
 
 $(window).load(function () {
-  var codesy = {user:{}}
-
-  Stripe.setPublishableKey('pk_test_NBTdQSXW5HcsEVNgzOZmmZng');
-
-  $('#refreshing').hide();
-
-  if ($("#codesy_user_id").length > 0) {
+    var codesy = {user:{}}
+    Stripe.setPublishableKey('pk_test_NBTdQSXW5HcsEVNgzOZmmZng');
```

Comment:
Note: that context processors will make **all** `settings.*` available to **all** templates, so it's handy to have around anyway.